components/multiselect/multiselect_list.tsx

```diff
@@ -137,12 +138,27 @@ export default class MultiSelectList extends React.Component<Props, State> {
                 className={rowSelected}
                 key={'multiselectoption' + option.value}
                 onClick={() => onAdd(option)}
+                onMouseMove={() => onMouseMove(option)}
             >
                 {option.label}
             </div>
         );
     }
 
+    private onMouseMove = (option: Value) => {
+        let i;
+        for (i = 0; i < this.props.options.length; i++) {
+            if (option === this.props.options[i]) {
+                break;
+            }
+        }
```

Comment:
wouldn't [indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) be a more concise way for this?